ansible/roles/docker/vars/main.yml

```diff
@@ -2,3 +2,4 @@
 # role variable for the swarm service
 
 docker_api_port: 2385
+thin_provisioned: false 
```

Comment:
this file has been removed in `ansible/master` as part of fixing https://github.com/contiv/ansible/issues/61, we will need to move the new variable in `roles/docker/defaults/main.yml` file